test/node-unit/cli/fixtures/bad-module.fixture.js

```diff
@@ -0,0 +1 @@
+throw new Error('this module is wonky');
```

Comment:
this has a final newline locally, but not sure if GitHub shows that.

Reply:
GH only shows when the final newline is missing.